src/backend/people/settings.py

```diff
@@ -451,6 +451,19 @@ class Base(Configuration):
         )
     )
 
+    # Feature flags
+    FEATURES_CONTACTS_DISPLAY = values.BooleanValue(
+        default=True, environ_name="FEATURES_CONTACTS_DISPLAY", environ_prefix=None
+    )
+    FEATURES_CONTACTS_CREATE = values.BooleanValue(
+        default=True, environ_name="FEATURES_CONTACTS_CREATE", environ_prefix=None
+    )
+    FEATURES_TEAMS_CREATE = values.BooleanValue(
+        default=True, environ_name="FEATURES_TEAMS_CREATE", environ_prefix=None
+    )
+    FEATURES_MAILBOXES_CREATE = values.BooleanValue(
+        default=True, environ_name="FEATURES_MAILBOXES_CREATE", environ_prefix=None
+    )
```

Comment:
I think we need a better name. I would have prefixed them with USER_ 
USER_MAILBOXES_CREATE ?

Reply:
And also what about the setting FEATURES below :thinking: